bot/exts/help_channels/_channel.py

```diff
@@ -80,7 +89,7 @@ async def send_opened_post_dm(thread: discord.Thread) -> None:
         try:
             message = await thread.fetch_message(thread.id)
         except discord.HTTPException:
-            log.warning(f"Could not fetch message for thread {thread.name} ({thread.id})")
+            log.warning(f"Could not fetch message for thread {thread.id}")
```

Comment:
I'm curious, why did we remove the name here ? Was it resulting in an AttributeError or is is just because the name can be too long and result in clutter ?

Reply:
Sentry groups similar alerts, adding the name meant it was different enough not to group, leading to a load of alerts

Reply:
Okay, makes sens

Reply:
This was part of the first commit, and it's for sentry. Basically in the case of a warning log, sentry doesn't get a traceback and only has the warning message to go off. Considering how much of the message itself is made up of the thread name, this leads to sentry failing to recognize and group these warnings as one thing. I'm not sure if this will fix it since the thread IDs are different as well of course, but I thought i'd try. Also having the thread name in the warning means the sentry issue name is really long, and doesn't display properly.